.dependabot/config.yml

```diff
@@ -1993,3 +1993,31 @@ update_configs:
         dependency_name: demisto/*
         update_type: all
 
+  - package_manager: python
+    directory: /docker/tidy
+    update_schedule: live
+    automerged_updates:
+    - match:
+        update_type: semver:minor
+  - package_manager: docker
+    directory: /docker/tidy
+    update_schedule: daily
+    automerged_updates:
+    - match:
+        dependency_name: demisto/*
+        update_type: all
+
+  - package_manager: python
+    directory: /docker/tidy123
```

Comment:
What is this one? I think this was added by mistake.

Reply:
Yes,  accident.

Removed it